<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotMonitorsTechTab</class>
 <widget class="QewExtensibleDialog" name="RobotMonitorsTechTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>504</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Recording and monitoring</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Velocities and position recording</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="fileCheckBox">
        <property name="text">
         <string>On</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel3_2">
        <property name="text">
         <string>File</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="vFileLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="vFileButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox2">
     <property name="title">
      <string>Velocities monitoring</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="monCheckBox">
            <property name="text">
             <string>On</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer17">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>310</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel2">
          <property name="text">
           <string>Monitor view</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="viewListBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentItem" stdset="0">
           <number>-1</number>
          </property>
          <item>
           <property name="text">
            <string>Oscilloscope</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paper</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="velsGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="4" column="0">
          <widget class="QLabel" name="TextLabel2_2_2">
           <property name="text">
            <string>N. of linear vel. labels</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0" rowspan="2">
          <widget class="QLabel" name="TextLabel1_5_2_3_3">
           <property name="text">
            <string>Minimun radial velocity</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabel1_5_2_2">
           <property name="text">
            <string>Time update (seconds)</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabel1_5_2_3">
           <property name="text">
            <string>Minimun linear velocity</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="spacer8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QDoubleSpinBox" name="minvel_linFloatSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>-100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="timeupdSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>30</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QSpinBox" name="vLinLabelsSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>11</number>
           </property>
           <property name="value">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <spacer name="spacer7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1" colspan="2">
          <spacer name="spacer7_2_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="4">
          <spacer name="spacer2_2_2_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <spacer name="spacer2_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="spacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="5" rowspan="2" colspan="3">
          <widget class="QLabel" name="TextLabel1_5_2_3_2_2">
           <property name="text">
            <string>Maximun radial velocity</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QLabel" name="TextLabel2_2_2_2">
           <property name="text">
            <string>N. of radial vel. labels</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="TextLabel2_2">
           <property name="text">
            <string>Number of time labels</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="5" colspan="2">
          <widget class="QLabel" name="TextLabel1_5_2_3_2">
           <property name="text">
            <string>Maximun linear velocity</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <spacer name="spacer7_2_2_2_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="spacer7_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2" rowspan="2" colspan="2">
          <widget class="QDoubleSpinBox" name="minvel_radFloatSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.100000000000364</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>-1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <spacer name="spacer2_2_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="7">
          <spacer name="spacer7_2_2_2_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="8" rowspan="2" colspan="2">
          <widget class="QDoubleSpinBox" name="maxvel_radFloatSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.009999999999991</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="8" colspan="2">
          <widget class="QDoubleSpinBox" name="maxvel_linFloatSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QSpinBox" name="tLabelsSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>9</number>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="4" column="9">
          <widget class="QSpinBox" name="vRadLabelsSpinBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>11</number>
           </property>
           <property name="value">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item row="0" column="6" colspan="3">
          <spacer name="spacer7_2_2_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="6" colspan="3">
          <spacer name="spacer7_2_2_2_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="spacer15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadFileButton">
       <property name="text">
        <string>Load from file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer15_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveFileButton">
       <property name="text">
        <string>Save to file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileCheckBox</tabstop>
  <tabstop>vFileLineEdit</tabstop>
  <tabstop>monCheckBox</tabstop>
  <tabstop>viewListBox</tabstop>
  <tabstop>timeupdSpinBox</tabstop>
  <tabstop>tLabelsSpinBox</tabstop>
  <tabstop>minvel_linFloatSpinBox</tabstop>
  <tabstop>maxvel_linFloatSpinBox</tabstop>
  <tabstop>minvel_radFloatSpinBox</tabstop>
  <tabstop>maxvel_radFloatSpinBox</tabstop>
  <tabstop>vLinLabelsSpinBox</tabstop>
  <tabstop>vRadLabelsSpinBox</tabstop>
  <tabstop>saveFileButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">/usr/include/qew5/qewextensibledialog.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
